notifications/final_job_status.go

```diff
@@ -0,0 +1,90 @@
+package notifications
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+
+	"github.com/textileio/powergate/v2/ffs"
+)
+
+type FinalJobStatus struct {
+	JobId      ffs.JobID
+	JobStatus  ffs.JobStatus
+	JobError   error
+	DealErrors []ffs.DealError
+}
+
+func (f FinalJobStatus) JobID() ffs.JobID {
+	return f.JobId
+}
+
+func (f FinalJobStatus) FinalUpdates() bool {
+	return true
+}
+
+func (f FinalJobStatus) MatchNotificationAlert(alert *ffs.WebhookAlert) bool {
+	return false
+}
+
+func (f FinalJobStatus) MatchNotificationEvent(event string) bool {
```

Comment:
Maybe we need to implement some Go linter in this project, but would be really useful to see comments on any exported type, func, var, const. Some thing are self descriptive, but some definitely are not, so comments are helpful.

Reply:
ok, will work on adding comments